geomstats/algebra_utils.py

```diff
@@ -157,3 +157,33 @@ def flip_determinant(matrix, det):
                 + (1. - mask)[..., None] * ones)
         return gs.einsum('...ij,...j->...ij', matrix, sign)
     return matrix
+
+
+def rotate_points(points, end_point):
```

Comment:
Should this be factorized in a module that implements fundamental actions of Matrix Lie Groups on Rn?